### PR TITLE
chore: add release automation

### PR DESCRIPTION
**Pull request by JonathanStoye**
- clean up .npmignore and dockerfiles
- add release automation

Signed-off-by: Jonathan Stoye <mail@jonathanstoye.de>

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,12 +1,10 @@
-Dockerfile
 .prettierrc
 postcss.config.js
 webpack.config.js
 .babelrc
 browserslist
 /node_modules
 .npmrc
-docs-image.docker
 /scripts
```
Comment (drinchev) [inline .npmignore]: You should add `/.travis.yml`, `/resources` and `/scripts` here, since we don't want them to be in the npm package.

Comment (JonathanStoye) [reply .npmignore]:
```suggestion
/scripts
/.travis.yml
/resources
```

File: scripts/tag-commit.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ $TRAVIS_COMMIT_MESSAGE =~ ^release ]]; then
```
Comment (jkettmann) [inline scripts/tag-commit.sh]: Do we need this if here? We already have this condition in the `deploy` script, inside `.travis.yml` right?

Comment (drinchev) [reply scripts/tag-commit.sh]: Unfortunately we do :( The deploy part in `travis.yml` actually expects to have `TRAVIS_TAG` set. It was a long process of *Trial/Error* yesterday until we reached that state ( https://github.com/lightelligence-io/react/commits/master/.travis.yml )

Comment (jkettmann) [reply scripts/tag-commit.sh]: Then I believe you :-)

**Pull request merged**